boards/nordic/nrf52840dk_opensk_a/build.rs

```diff
@@ -24,6 +24,11 @@ static mut STORAGE_LOCATIONS: [kernel::StorageLocation; 5] = [
         storage_type: kernel::StorageType::Store,
     },
     // Partitions can also be split to maximize MPU happiness.
+    kernel::StorageLocation {
+        address: 0x4000,
+        size: 0x1000,
+        storage_type: kernel::StorageType::Partition,
```

Comment:
I guess we can't merge those 2 locations because of MMU? Same for the 2 locations below.

Reply:
We can actually for metadata, just have to assume some knowledge inside Env. The other partitions not so much, since MPU restrictions.